test/tests/AelinUpFrontDeal.purchase.t.sol

```diff
@@ -1397,10 +1316,6 @@ contract AelinUpFrontDealPurchaseTest is Test, AelinTestUtils, IAelinUpFrontDeal
         emit AcceptDeal(user2, purchaseCollection2, purchaseCollection2, poolSharesAmount, poolSharesAmount);
         AelinUpFrontDeal(dealAddressNftGating721).acceptDeal(nftPurchaseList, merkleDataEmpty, purchaseCollection2);
 
-        // user2 is now blacklisted from purchasing again in this collection
-        vm.expectRevert("wallet already used for nft set");
```

Comment:
@CookedCookee when it's an  ERC721 collection I guess we can replace this verification with `tokenId already used` instead as we want to make sure users can't buy again with the tokenIDs they just used